Source/ACE.Database/CharacterDatabase.cs

```diff
@@ -49,7 +49,13 @@ private enum CharacterPreparedStatement
             CharacterPropertiesIntInsert,
             CharacterPropertiesBigIntInsert,
             CharacterPropertiesDoubleInsert,
-            CharacterPropertiesStringInsert
+            CharacterPropertiesStringInsert,
+
+            CharacterPropertiesBoolUpdate,
+            CharacterPropertiesIntUpdate,
+            CharacterPropertiesBigIntUpdate,
+            CharacterPropertiesDoubleUpdate,
+            CharacterPropertiesStringUpdate
         }
```

Comment:
Also, in future code too save skills, we can duplicate this work for the other databases, then `properties`.